From 715b05f97cde12424bb6d425264569f7f921dc72 Mon Sep 17 00:00:00 2001
From: Jan Macku <jamacku@redhat.com>
Date: Tue, 7 Nov 2023 13:33:40 +0100
Subject: [PATCH] ci: add missing permissions

issues: write is required for the pull request merging according to:

https://github.com/cli/cli/discussions/6379#discussioncomment-3806051

rhel-only

Related: RHEL-1086
---
 .github/workflows/source-git-automation-on-demand.yml | 1 +
 .github/workflows/source-git-automation.yml           | 1 +
 2 files changed, 2 insertions(+)

diff --git a/.github/workflows/source-git-automation-on-demand.yml b/.github/workflows/source-git-automation-on-demand.yml
index 2c506f2b3e..bf2ea2260c 100644
--- a/.github/workflows/source-git-automation-on-demand.yml
+++ b/.github/workflows/source-git-automation-on-demand.yml
@@ -63,6 +63,7 @@ jobs:
       contents: write
       statuses: write
       checks: write
+      issues: write
       pull-requests: write
 
     steps:
diff --git a/.github/workflows/source-git-automation.yml b/.github/workflows/source-git-automation.yml
index 17135b590f..d71664efa0 100644
--- a/.github/workflows/source-git-automation.yml
+++ b/.github/workflows/source-git-automation.yml
@@ -92,6 +92,7 @@ jobs:
     permissions:
       contents: write
       checks: write
+      issues: write
       pull-requests: write
 
     steps:
